misc/userscripts/qute-pass

```diff
@@ -17,20 +17,21 @@
 # You should have received a copy of the GNU General Public License
 # along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
 
+"""
+Insert login information using pass and a dmenu-compatible application (e.g. dmenu, rofi -dmenu, ...). A short
+demonstration can be seen here: https://i.imgur.com/KN3XuZP.gif.
```

Comment:
I really like the demonstration, I think more things should have stuff like this!

Reply:
A picture is worth a thousand words... and a GIF well, `frames * 1000` I suppose :smile:.